README.md

```diff
@@ -141,6 +129,31 @@ jobs:
         asset_content_type: application/zip
 ```
 
+### Unprivileged User Workflow
+
+If you need to run take action as an unprivileged user, the `AS_USER` variable
+[documented above](#unprivileged-user) will not work, both because GitHub
+actions [require the root user] to work with the workspace and because GitHub
+does not execute `entrypoint` script.
+
+Instead, set things up as the root user, then use [gosu] to execute a command as
+the `postgres` (can still run `sudo`) or `nobody` (no privileges at all) user.
```

Comment:
```suggestions
Note: GitHub Actions do not support the `AS_USER` facility when using this image
because they do not execute the `entrypoint` script.

If you _do_ need to take action as a lower-privileged user in a GitHub Action (such
as running the `initdb` command in a test script), you can use `gosu` in a `run:` block to execute the given task as either the `postgres`  or `nobody` user.
```

Reply:
I think we were both editing this at the same time. The changes in 1b4556c fix the issue I think.